src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -4,6 +4,23 @@
+    0 0 0 1px var(--g-color-line-generic-solid),
+    0 3px 10px 0 var(--g-color-sfx-shadow),
+    0 8px 20px -4px var(--g-color-sfx-shadow);
+$floating-surface-box-shadow-dark:
+    0 0 0 1px var(--g-color-line-generic-solid),
+    0 4px 12px 0 var(--g-color-sfx-veil),
+    0 10px 24px -2px var(--g-color-sfx-shadow-heavy);
+
+@mixin floating-surface-box-shadow-default {
+    --_--floating-surface-box-shadow-default: #{$floating-surface-box-shadow-light};
+
+    :global(.g-root_theme_dark) &,
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)